shared/actions/config/index.js

```diff
@@ -119,14 +135,15 @@ const bootstrap = (opts?: BootstrapOptions = {}): AsyncAction => (dispatch, getS
       console.log('[bootstrap] bootstrapping on connect')
       dispatch(bootstrap())
     })
-    engine().listenOnDisconnect('daemonError', () => {
```

Comment:
have to move this down else we flash the disconnect when we actually disconnect between retries